### ISSUE TITLE
I cant some nether update blocks

### ISSUE THREAD

**Issue by DesertSoldier43** [PowerNukkit/PowerNukkit#898]
Hallo, so recently I am making a server and I want to add a nether block, but I can't. I place it and it disappears. Is there any way to fix it or is there a plugin? Some nether blocks are the respawn anchor, shroomlight, lodestone, and maybe some others I don't know.

**Comment by MonoAdrian23** [comment]
All new blocks are supported only in 1.4.0.0 PowerNukkit.

**Comment by DesertSoldier43** [comment]
> Hey @DesertSoldier43, thank you for the report!
> 
> ⚠️ **I could not find which PowerNukkit version you are using from your message.** ⚠️
> 
> Please, run the command: `/debugpaste` it will generate a link, send us this link replying this issue.
> It's an important step to get your issue resolved.
> 
> Don't say _latest_ or _current version_, we support different versions and your latest may not be the same latest as our, and your latest will not be the same latest in the future, so please, use a fixed version.
> 
> Thank you 👍

https://hastebin.com/ufijokupim

**Comment by DesertSoldier43** [comment]
> All new blocks are supported only in 1.4.0.0 PowerNukkit.

But isn't there a bug where if there are multiple people online and if another person joins, it kicks pretty much all?

**Comment by MonoAdrian23** [comment]
Yes, please wait for a fix or try to find a Bleeding Version without this bug

**Comment by DesertSoldier43** [comment]
> Yes, please wait for a fix or try to find a Bleeding Version without this bug

Ok, thanks I will wait. Sorry if it seemed like I was cranky, it is midnight rn. not to be rude

**Comment by GoodLucky777** [comment]
It is implemented in 1.4-PN
